lib/google_places/rectangle.rb

```diff
@@ -0,0 +1,19 @@
+module GooglePlaces
+  class Rectangle
+    def initialize(lat1, lng1, lat2, lng2)
```

Comment:
I wanted to leave the option option to create the rectangle from any start+end point combination -- NE + SW, SE + NW, NW + SE, and so on.  I was afraid that specifying SE/NW would infer to the user that they could only define bounds by that given pair.  Do you think start_lat, start_lng, end_lat, end_lng would gain us some clarity?